src/runtime/basic-runtime-test.js

```diff
@@ -274,6 +232,12 @@ describes.realWin('BasicRuntime', {}, (env) => {
       // PageConfigResolver should have been created and attempted to resolve
       // the PageConfig.
       expect(resolveStub).to.be.calledOnce;
+
+      // Default metering handler in entitlements-manager should be enabled
+      // for BasicRuntime.
+      expect(
+        configuredRuntime.entitlementsManager().enableDefaultMeteringHandler_
```

Comment:
Can you rewrite these assertions so they do not check a private variable? (ie, test behavior instead of implementation).

Reply:
Done